mos-platform/cpm65/clang.cfg

```diff
@@ -0,0 +1,3 @@
+-D__CPM65__
```

Comment:
You might also want to set `-mlto-zp` to let the compiler know how many zero page locations it's allowed to play with, not including the imaginary register bank. This allows it to lift global variables, callee-saved-registers, and local variables to the zero page, if it's profitable to do so.

Reply:
I did try this, but it ended up producing bad code and my programs didn't work. I'm still investigating why.